src/libraries/token/VariableSupplyToken.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/token/StandardToken.sol';
+
+
+contract VariableSupplyToken is StandardToken {
+    using SafeMath for uint256;
+
+    event Mint(address indexed target, uint256 value);
+    event Burn(address indexed target, uint256 value);
+
+    function mint(address _target, uint256 _amount) internal returns (bool) {
```

Comment:
This function (and the next one) are not returning boolean values as specified in their function declarations.

Also, can you please add a TODO or FIXME to the test_dispute_bond_token.py file so we remember to verify that these new functions work properly?